### ISSUE TITLE
Sync doesn't work sometimes

### ISSUE THREAD

**Issue by glow12121** [appml/nano#11]
Synchronization between Neutrinote's repository and mirror - and a Dropbox cloud doesn't work at times.

For example,  in the previous message I described how I wiped all notes on my phone and copied them again. Before doing this, I created a dummy note called First.md. At 4 pm I deleted First.md on my PC. It was deleted from the cloud. Dropsync on Android deleted the note a few minutes later. But First.md is still on Neutrinote's list, *almost four hours later.*

I checked the repository and mirror folders. First.md is not in the mirror folder but it is in the repo folder.

Something is wrong here.

**Comment by appml** [comment]
Try pull to refresh to issue a sync between repo and mirror.

On the other hand, may I ask the make of your device?  Are you sure DropSync is not seeing neutriNote's repo folder?  Even though an app is not supposed to access the folders of other apps.  Some manufacturers seem to allow that.

**Comment by appml** [comment]
Also please make sure your repo is at the default location.
![Screenshot_20220621-193247](https://user-images.githubusercontent.com/7504229/174913483-74d73dad-8293-47f0-a053-632aa9e7af77.png)

**Comment by glow12121** [comment]
Pulling to refresh didn't help, tried it.

I use a pretty old Xiaomi phone that still works well with Lineage OS 18.1 (Android 11-based). You are right, I was able to navigate Dropsync to Neutrinote's repository within Android folder. Now things are smoother and more responsive.

I also disabled back ups & syncing as they are not necessary with a cloud.

I use /files folder as the repository. In fact, I cannot change it. When I tap, this balloon appears below, that is all.

![IMG_20220622_031435](https://user-images.githubusercontent.com/29035180/174919628-246658d8-f788-4216-b974-18c5a3e541da.jpg)
